environment.yml

```diff
@@ -7,8 +7,8 @@ dependencies:
   - numpy=1.21
   - astropy~=4.2
   - pip
-  - ctapipe~=0.11
-  - gammapy=0.18
+  - ctapipe=0.12
+  - gammapy=0.19
```

Comment:
what about updating `matplotlib` and `pyparsing`?

Reply:
yes, done